packages/components/src/Drawer/Drawer.scss

```diff
@@ -46,15 +48,36 @@ $tc-action-bar-background-color: $concrete !default;
 	height: 100%;
 }
 
+.tc-drawer-tabs-container {
+	background-color: $tc-drawer-tabs-background;
+}
+
 .drawer-stacked {
 	width: 100%;
 }
 
+@media screen and (min-width: $screen-xs-max) {
+	.drawer-stacked {
+		.tc-drawer-header-title,
+		.tc-drawer-tabs,
+		.tc-drawer-content,
+		.tc-drawer-actionbar,
+		.tc-drawer-footer > nav {
+			width: $tc-drawer-content-max-width;
+			margin: 0 auto;
+		}
+	}
+}
+
 .tc-drawer-header {
-	padding: $padding-normal;
 	background-color: $brand-primary-lighter;
 
-	h1 {
+	.tc-drawer-header-title {
+		padding: $padding-normal;
+		position: relative;
```

Comment:
Why is this relative?

Reply:
@frassinier because 'cancel' cross button has position absolute.